Riva0110/Vocab-Garden
# Feature/review battle

### Description

* Add battle mode for review feature

### Test Plan

* Check the answers and score have shown correctly
* Check if turning to next round every 5 seconds or when both of users has answered
* Check both of users can see the same information
